src/components/svg.js

```diff
@@ -0,0 +1,255 @@
+import React, { PureComponent, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import * as d3 from 'd3'
+import * as actions from '../actions/actionIndex'
+import reactCSS from 'reactcss'
+
+const styles = reactCSS({
+  'default': {
+    svgMain: {
+      height: '280px',
+      marginBottom: '0',
+    }
+  }
+})
+
+const { number, object, func} = PropTypes
+
+class Svg extends PureComponent {
```

Comment:
I think we talked about this component during our session earlier today. So i'll wait to comment on this component until after your latest changes are committed/pushed.